### PR TITLE
Create Models and Foreign Keys

### PR DESCRIPTION
**Pull request by Solobarine**
This is a required pull request to review the following changes made to the blog project:
-
The models were created.
Foreign keys were added.
Migrations were created and deployed to the darabase.

### CODE REVIEW

**Review by andersonlebon — COMMENTED** [review]
## Hi @Solobarine 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a professional Readme file kindly follow this [microverse template](https://github.com/microverseinc/readme-template) and work on your README file the request another review

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,24 +1,133 @@
...
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank everyone
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (rotimiazeez) [inline README.md:128]: Please not that the MIT License link provided is invalid. Kindly ensure a valid link is provided 🙂
Screenshot_20230330_073416

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @Solobarine👍🏽,
## Good job so far!
### To Highlight 👇🏽
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Good commit message(s) 😉
- Descriptive README 📖
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

**Review by Ismailco — APPROVED** [review]
### Hi @Solobarine, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️ No linter errors
✔️ Work documented professionally

## Optional suggestions
- [ ] Kindly make sure to pay attention to the workflow the next time, this project should use `Gitflow`, not `Github Flow`. That means you should create a dev branch for development, and you merge the final version of the app to the main when it's finished

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Solobarine** [comment]
Thank you @Ismailco, I will try to implement it next time.

**Pull request merged**